images/git/image.yaml

```diff
@@ -1,36 +1,10 @@
-# Opt-out of the new terraform-based release
-terraform: false
-
 versions:
```

Comment:
I think we're supposed to end up with one item in this list. Is there a reason this has to have an item for each yaml file?